db.helpers.js

```diff
@@ -0,0 +1,26 @@
+const knex = require('knex');
+
+const dbConfig = require('./knexfile.js');
+const db = knex(dbConfig.development);
+
+module.exports ={
+    getDishes,
+    addDish,
+    getDish,
+    getRecipes,
+    addRecipe
+}
+
+function getDishes(){
+    return db('dishes');
+}
+
+function addDish(dish){
+    return db('dishes').insert(dish);
+}
+
+function getDish(id){
+    return db('dishes')
+        .where('dishes.id', id)
+        .join('recipes', 'dishes.id', 'recipes.dish_id')
```

Comment:
You could make this a little shorter with aliases:
```javascript
return db('dishes as d')
  .where('d.id', id)
  .join('recipes as r', 'd.id', 'r.dish_id');
```
I also question whether or not you need to specify recipes as the table for `dish_id` since `dishes` _probably_ doesn't have that as a column. If that's right, then just:
```javascript
return db('dishes as d')
  .where('d.id', id)
  .join('recipes as r', 'd.id', 'dish_id');
```
is good.